### PR TITLE
 Adding a New VRF

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
@kevin-hereworks

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 1]
```diff
@@ -19,6 +19,14 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_DFMSC_DSP_ZTC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:22]: There are no abstract types here to capture the actual cooling control (e.g., DXDC, DXZC, etc.)

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -19,6 +19,14 @@
...
+  description: "Variable refrigerant flow unit with zone cooling and heating control, discharge fan speed modes"
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:23]: please add new GUID

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 1 above)
Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:22]: Hi @Ciaran-Oflannagain we encourage you to model the device as it exists in real life (not how it is digitally represented) -- we know that this has a DX control of some type (even if the fields are not available for integration). Based on what you already have, I would suggest adding DXZTC and marking the required fields as MISSING in your building config model.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: @cstirdivant - thanks for your input I have added DXZTC and will make a note in building config. Thanks, Ciaran

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**